outmon/vertexmon.go

```diff
@@ -190,11 +192,27 @@ func (vm *vertexMonitor) printError() bool {
 	switch {
 	case reErrExitCode.MatchString(errString):
 		m := reErrExitCode.FindStringSubmatch(errString)
-		errString = fmt.Sprintf(""+
-			"      The%s command\n"+
-			"          %s\n"+
-			"      did not complete successfully. Exit code %s",
-			internalStr, indentOp, m[2])
+
+		// Ignore the Error, default case will print it as a string using the source, so we won't miss any data.
+		exitCode, _ := strconv.ParseUint(m[2], 10, 32)
+
+		switch exitCode {
+		case 137:
+		case math.MaxUint32:
```

Comment:
I just noticed that this should be `case 137, mathUint32:` (otherwise it does nothing on 137).